apps/cloudinary/src/index.js

```diff
@@ -162,10 +187,28 @@ setup({
     {
       "id": "startFolder",
       "name": "Starting folder",
-      "description": "Path to starting folder for the media library dialog",
+      "description": "A path to a folder, which the Cloudinary Media Library will automatically browse to on load",
       "type": "Symbol",
       "required": false,
       "default": ""
+    },
+    {
+      "id": "quality",
+      "name": "Media Quality",
+      "description": "The quality level of your assets. This can be a fixed number ranging from 1-100, or you can get Cloudinary to decide the most optimized level by setting to 'auto'. More available options are auto:low/auto:eco/auto:good/auto:best. If you wish to use the original level, set to 'none'",
+      "type": "List",
+      "value": "auto,none,auto:low,auto:eco,auto:good,auto:best,10,20,30,40,50,60,70,80,90,100",
+      "required": true,
+      "default": "auto"
+    },
+    {
+      "id": "format",
+      "name": "Format",
+      "description": "The format of the assets. This can be set manually to a specific format - 'jpg' as an example (all supported formats can be foudn here - https://cloudinary.com/documentation/image_transformations#supported_image_formats. By setting to 'auto' Cloudinary will decide on the most optimized format for your users. If you wish to keep the original format, set to 'none'", 
```

Comment:
```suggestion
      "description": "The format of the assets. This can be set manually to a specific format - 'jpg' as an example (all supported formats can be found here - https://cloudinary.com/documentation/image_transformations#supported_image_formats. By setting it to 'auto', Cloudinary will decide on the most optimized format for your users. If you wish to keep the original format, set it to 'none'.", 
```